Library/Formula/dependency-check.rb

```diff
@@ -26,13 +26,14 @@ def install
     corejar = libexec/"repo/org/owasp/dependency-check-core/#{version}/"\
       "dependency-check-core-#{version}.jar"
     system "unzip", "-o", corejar, "dependencycheck.properties", "-d", \
-      etc/"dependencycheck"
-    libexec.install_symlink etc/"dependencycheck" => "etc"
+      libexec/"etc"
+    etc.install_symlink libexec/"etc/dependencycheck.properties" => \
+      "dependencycheck/dependencycheck.properties"
   end
 
   test do
     output = `#{libexec}/bin/dependency-check --version`.strip
-    assert_equal("Dependency-Check Core version 1.2.11", output)
+    assert_match("Dependency-Check Core version ", output)
```

Comment:
Yeah - was going to do that, but it was suggested by @mikemcquaid that I leave the version out altogether. Will put it in.